src/services/WaveformDataUtils.js

```diff
@@ -3,9 +3,38 @@ import StructuralMetadataUtils from './StructuralMetadataUtils';
 // Colors for segments from Avalon branding pallette
 const COLOR_PALETTE = ['#80A590', '#2A5459', '#FBB040'];
 
+// Constants used to generate dummy waveform data when actual waveform.json is missing
+const SAMPLE_RATE = 44100,
+  BITS = 8,
+  SAMPLES_PER_PIXEL = 183,
+  MAX_VALUE = 1,
+  MIN_VALUE = -1;
```

Comment:
I have mild concern about generating waveform data on the client side because these settings might get out of sync with the backend waveform settings. If it doesn't affect functionality (zoom level, granularity of timestamp, etc) then I guess it's ok. 

What do you think about generating data on the backend? The pros would be the client doesn't care if it's empty or not.

Reply:
I checked this by changing values and what I observed was that, the zoom level (time shown in the zoomed waveform) changed when the sample rate and samples per pixel values were changed. I'm not sure whether this will break the waveform.
If these value pairs have a standard then we might be okay. I guess I fully don't understand the mathematics of how this works.
Yeah, the best solution would be to let the back-end handle this so that there's no odd looking waveforms in the future :+1: